src/parent/reducers/reducers.js

```diff
@@ -1,12 +1,61 @@
 import { combineReducers } from 'redux';
+import reduceReducers from 'reduce-reducers';
+
+import { PARENT, SIDEBAR } from '../../shared/constants/actionTypes';
 
 import childWindows from './childWindows';
+import closedWindows from './closedWindows';
 import dragOut from './dragOut';
 
-const rootReducer = combineReducers({
-    childWindows,
-    dragOut
-});
+const rootReducer = reduceReducers(
+    combineReducers({
+        childWindows,
+        closedWindows,
+        dragOut
+    }),
+    (state, action) => {
+        switch (action.type) {
+        case SIDEBAR.REOPEN_WINDOW: {
+            const newClosedWindow = Object.assign({}, state.closedWindows[action.reopenWindowName], {});
+
+            delete newClosedWindow.date;
+
+            const newClosedWindows = Object.assign({}, state.closedWindows);
+            delete newClosedWindows[action.reopenWindowName];
+
+            return {
```

Comment:
Shouldn't this also handle the `dragOut` part of the state? Otherwise won't the state be returned without the `dragOut` slice of state?